README.md

```diff
@@ -1,3 +1,149 @@
 # Image Build
 
 Monorepo menagerie of container images and associated build automation
+
+## Podman / Buildah / Skopeo
+
+## Overview
+
+[comment]: <> (***ATTENTION*** ***WARNING*** ***ALERT*** ***CAUTION*** ***DANGER***)
+[comment]: <> ()
+[comment]: <> (ANY changes made below, once committed/merged must)
+[comment]: <> (be manually copy/pasted -in markdown- into the description)
+[comment]: <> (field on Quay at the following locations, where * represents podman|buildah|skopeo:)
+[comment]: <> ()
+[comment]: <> (https://quay.io/repository/containers/*)
+[comment]: <> (https://quay.io/repository/*/stable)
+[comment]: <> (https://quay.io/repository/*/testing)
+[comment]: <> (https://quay.io/repository/*/upstream)
+[comment]: <> ()
+[comment]: <> (***ATTENTION*** ***WARNING*** ***ALERT*** ***CAUTION*** ***DANGER***)
+
+These directories contain the Containerfiles necessary to create the images housed on
+quay.io under the their namespace in addition to the 'containers' namespace.  These
+images public and can be pulled without credentials.  These container images are secured and the
+resulting containers can run safely with or without privleges.
+
+The container images are built using the latest Fedora and then the respective tools are installed.
+The `$PATH` in the container images is set to the default provided by Fedora.  Neither the
+`$ENTRYPOINT` and the `$WORKDIR` variables are set within these container images, and as
+such they default to `/`.
+
+The container images are tagged as follows, where `*` represents either `podman`, `buildah`
+or `skopeo`:
```

Comment:
This is an IDK question.  Is Brent cooking up a new machine image and should, or maybe will be, included here too?

Reply:
That's my hope for the future.  It makes a lot of sense for several reasons to put both the image content and build automation close.